src/modules/EventList/index.tsx

```diff
@@ -327,7 +327,7 @@ export const EventList = (_props: EventListProps): ReactElement => {
             listType={state.listType}
             onBackFromTranscript={(event) => onSelectEvent(event, { value: null })}
             onCompanyChange={onSelectCompany}
-            onSearchChange={({ currentTarget: { value } }) => setState({ ...state, searchTerm: value })}
+            onSearchChange={(_e, { value }) => typeof value === 'string' && setState({ ...state, searchTerm: value })}
```

Comment:
I _think_ this can now just be `onSearchChange={handlers.searchTerm}`, one of the nice things about having this all use the same interface is that the utilities around it will now work too,